app/src/main/java/com/mobile/tandil/kotlinbaseproject/database/ReservationDataBase.kt

```diff
@@ -0,0 +1,17 @@
+package com.mobile.tandil.kotlinbaseproject.database
+
+import com.mobile.tandil.kotlinbaseproject.entity.Reservation
+
+class ReservationDataBase {
+    private var reservations: HashMap<Int, List<Reservation>> = hashMapOf()
+    private var reservationList: List<Reservation> = emptyList()
```

Comment:
you do not need this, because in this way you will save all the reservations without matter the parking lot